Resources/config/assetic_injector.json

```diff
@@ -4,7 +4,10 @@
         "javascripts":
         {
             "foot": "@TroopersAjaxBundle/Resources/public/js/ajax.js",
-            "bootstrap-extra": "@TroopersAjaxBundle/Resources/public/js/ajax-modal.js"
+            "bootstrap-extra": [
+                "@TroopersAjaxBundle/Resources/public/js/ajax-modal.js",
+                "@TroopersAjaxBundle/Resources/public/js/ajax-updateOnChange.js"
```

Comment:
I don't think ajax-updateOnChange.js is related to bootstrap-extra. I guess foot is a better tag.